frigate/util/image.py

```diff
@@ -765,7 +765,7 @@ def add_mask(mask: str, mask_img: np.ndarray):
 
 
 def get_image_from_recording(
-    file_path: str, relative_frame_time: float
+    file_path: str, relative_frame_time: float, codec: str, height: int
```

Comment:
might be better to make height optional and pass `None`

Reply:
yep